SWLOR.Game.Server/Feature/ChatCommandDefinition/DMChatCommand.cs

```diff
@@ -406,10 +406,66 @@ private void SetLocalVariable()
 
                     SendMessageToPC(user, "Local string set: " + variableName + " = " + value);
                 });
+
+            _builder.Create("settag")
+                .Description("Sets a local tag on a target.")
+                .Permissions(AuthorizationLevel.DM, AuthorizationLevel.Admin)
+                .AvailableToAllOnTestEnvironment()
```

Comment:
I don't want this exposed to all players on the test server. SetTag is a pretty dangerous function to give to DMs, let alone players. Definitely remove this line please.

Reply:
Oh, yeah, true. Removing.